websites/Z/Zybooks/presence.ts

```diff
@@ -0,0 +1,99 @@
+const presence = new Presence({
+	clientId: "1152294065115779102",
+});
+
+function getTextContent(querySelectorString: string): string {
+	return (
+		document
+			.querySelector<HTMLElement>(querySelectorString)
+			?.textContent?.trim() || ""
+	);
+}
+
+const zybooksTimestamp: {
+	hash: string;
+	timestamp: number;
+} = {
+	hash: "",
+	timestamp: Date.now(),
+};
+
+presence.on("UpdateData", async () => {
+	const { pathname, href } = document.location;
+	let hash: string = href;
+	const presenceData: PresenceData = {
+		startTimestamp: zybooksTimestamp.timestamp,
+		largeImageKey: "https://i.imgur.com/I4RTS73.png",
```

Comment:
Here a square 512x512 logo
```suggestion
		largeImageKey: "https://i.imgur.com/1lSQQh1.png",
```

Reply:
Thank you!